src/commands/dev.ts

```diff
@@ -220,7 +224,8 @@ export default class Dev extends BaseCommand {
     }
 
     const interfaces_map = DeployUtils.getInterfacesMap(flags.interface);
-    const component_secrets = DeployUtils.getComponentSecrets(flags.parameter, flags.secrets);
+    const component_secrets = DeployUtils.getComponentSecrets(flags.secret, flags['secret-file']);
+    const component_parameters = DeployUtils.getComponentSecrets(flags.parameter, flags['secret-file']);
```

Comment:
Does this need to be removed or still keeping it for the depreciated parameters?

Reply:
Yeah we're keeping it for now to be removed at a later time

https://gitlab.com/architect-io/architect-cli/-/issues/404